src/promptflow/promptflow/contracts/multimedia.py

```diff
@@ -63,3 +62,28 @@ def serialize(self, encoder: Callable = None):
         if encoder is None:
             return self.__str__()
         return encoder(self)
+
+
+class Text(str):
+    def __new__(cls, value: str, annotations: list = None):
+        obj = str.__new__(cls, value)
+        obj._annotations = annotations
+        return obj
+
+    @classmethod
+    def deserialize(cls, data: dict):
+        """Deserialize the dictionary to the text object."""
+
+        text = data.get("text", "")
+        if isinstance(text, dict):
+            return cls(value=text.get("value", ""), annotations=text.get("annotations", []))
+        else:
+            return cls(value=text)
+
+    def serialize(self):
+        """Serialize the text to a dictionary."""
+
+        if self._annotations is None:
+            return {"type": "text", "text": self}
```

Comment:
`self` is a class instance. How about something like `self.value` here?

Reply:
Text is a derivative of str, this implementation method has been tested to be feasible.